index.html

```diff
@@ -21,17 +21,22 @@
       <a href="index.html" class="logo">
         <img src="./assets/img/logo.svg" alt="Joshua Ivie">
       </a>
-      <div id="menuToggle" class="menu-toggle">
-        <span></span>
-        <span></span>
-        <span></span>
-      </div>
-      <div class="navigation-menu">
-        <ul>
-          <li><a href="#portfolio">Portfolio</a></li>
-          <li><a href="#about">About</a></li>
-          <li><a href="#contact">Contact</a></li>
-        </ul>
+      <div class="menu-wrapper" id="menuWrapper" onclick="menuMob()">
+        <div class="menu-overlay"></div>
+        <div class="menu-content">
+          <div id="menuToggle" class="menu-toggle">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="navigation-menu">
+            <ul>
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
+            </ul>
+          </div>

```

Comment:
- [ ] Just two concerns that need to be addressed before moving on:
  - When the first loads, the menu window is being rendered, <= it should not be like that, it should be the content of the page loaded instead,
  - When clicking on the contact menu, the page should scroll down to the `Contact me` section, kindly address these comments and submit another review request. 

At your best, please watch the following screencast for more context

https://user-images.githubusercontent.com/56886064/150879544-23b1fd1b-91f6-4a69-a1c7-3d108a4e388e.mp4

Reply:
Thank you for your review @descholar-ceo. The issues you raised have now been fixed.